leaf-ui/js/onFunctions.js

```diff
@@ -780,7 +766,10 @@ graph.on("add", function(cell) {
 		cell.attr('.funcvalue/text', ' ');
 
 	} else if (cell instanceof joint.shapes.basic.Actor) {
-		createActor(cell);
+        createdInstance = createdInstance.filter(view => view.model instanceof joint.shapes.basic.Actor);
```

Comment:
Looks awesome. Add a comment for what this is doing, and make an issue for the potential duplicate names (if you make Actor_0, Actor_1, and then delete Actor_0 and make a new actor then you will have two Actor_1s for example)